ci-operator/config/openshift/console/openshift-console-master.yaml

```diff
@@ -43,3 +46,40 @@ tests:
   commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-frontend.sh
   container:
     from: test-bin
+
+#  builder_run 'GUI-Tests' 1 ./test-gui.sh crud
+- artifact_dir: /tmp/artifacts
+  as: GUI-Tests
+  commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-gui.sh crud
+  container:
+    from: test-bin
+
+#  builder_run 'GUI-Tests-New-App' 1 ./test-gui.sh newApp
+- artifact_dir: /tmp/artifacts
+  as: GUI-Tests-New-App
+  commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-gui.sh newApp
+  container:
+    from: test-bin
+
+#  builder_run 'GUI-Tests-OLM' 1 ./test-gui.sh olm
+- artifact_dir: /tmp/artifacts
+  as: GUI-Tests-OLM
+  commands: export ARTIFACT_DIR=/tmp/artifacts; ./test-gui.sh olm
```

Comment:
This requires OLM be installed on the cluster.

Reply:
@ecordell @smarterclayton what's the status of this? Do we have documents on what people can expect from these test harnesses?

Reply:
OLM is installed by default - not sure what the question is, really.